.github/PULL_REQUEST_TEMPLATE/bug_PR.md

```diff
@@ -0,0 +1,101 @@
+---
+name: Bug PR
+about: Submit a PR for a bug
+title: 'BUG: '
+labels: bug
+assignees: ''
+
+---
+### About this repo
+- `q2-moshpit` is a `qiime2` plugin that offers various analyses for shotgun metagenome data.
+- <!--- What part of the repo is the current PR concerned with? E.g. The current PR addressed a bug in action X, which runs a Y analysis on the inputs. -->
+
```

Comment:
```suggestion
```
We discussed that an about this repo part should be removed because it does not belong here. Also the description of the bug should already be in the issue.